app/scripts/modules/core/src/managed/artifactDetail/constraints/constraintRegistry.tsx

```diff
@@ -137,7 +137,7 @@ const constraintsRenderer: { [type in IConstraint['type']]?: IConstraintRenderer
         case 'OVERRIDE_FAIL':
           return (
             <span>
-              <span>Manually {status === 'OVERRIDE_PASS' ? 'approved' : 'rejected'}</span>{' '}
+              <span>Manually {constraint.status === 'OVERRIDE_PASS' ? 'approved' : 'rejected'}</span>{' '}
```

Comment:
I guess there's a `window.status` field, which is why Typescript doesn't complain about this?

Reply:
Yeah :(
I opened a PR for that a couple of hours ago

Reply:
https://github.com/spinnaker/deck/pull/9010

Reply:
ha

Reply:
I'll close this and let you deal with the other thing if you want